app/templates/main/validation-errors.html

```diff
@@ -17,18 +19,32 @@
 {% set mainClasses = "govuk-!-width-full" %}
 
 {% block content %}
-    <div class="upload-data-container govuk-!-margin-top-5">
-        <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data">
+    <div class="upload-data-container govuk-!-margin-top-5" id="main_container">
+        {% if pre_error %}
+            {{ errorSummary(pre_error) }}
+        {% endif %}
+        <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data" id="submit_form">
```

Comment:
Is this change descoped?

Reply:
Good spot, I totally missed these changes